docs/source/workbench/instrumentviewer.rst

```diff
@@ -7,61 +7,58 @@ Instrument Viewer Widget
 Overview
 --------
 
-Mantid visualisation system allows users to view the geometry of an instrument and also select and query individual detectors.
-An instrument is always shown in the context of a workspace containing data recorded with this instrument.
+The Mantid visualisation system allows users to view the geometry of an instrument and also select and query individual detectors.
+An instrument is always shown in the context of a workspace, which contains data recorded with this instrument.
 The detectors are coloured according to the data in the corresponding spectra of the workspace.
 The user can select individual detectors by clicking on them and request to see the data recorded by this detector as a table or a graph.
 The instrument view also allows users to see close-ups of any component.
 
 
 Instrument View Pane
 --------------------
-The instrument view window can be obtained by right clicking on the workspace of interest and selecting "Show instrument".
+The instrument view window can be obtained by right clicking on the workspace of interest and selecting ``Show Instrument``.
 A 2D projection of the detector arrangement will be visible with each detector pixel color coded depending on the integrated number of counts in its corresponding spectrum, as in the image below.
```

Comment:
A mixture of color and colour is used in this document. Whilst matplotlib uses the American spelling and so specific references to matplotlib features need to be spelled correctly in relation to that I think we need some consistency when using these words outside that context. In this case can we change `detector pixel color` to `detector pixel colour`?

Reply:
Are we not changing this one?

Reply:
Ah I see we're going with American spelling throughout so ignore my comment here.

Reply:
Yeah, it'd just end up horrible with the mix of mpl and our own code that uses the US spelling.